backend/service/planMembership.ts

```diff
@@ -0,0 +1,10 @@
+import PlanMembershipModel from '../models/planMembership';
+
+export default class PlanMembership {
+  public static async initiatePlanMembership (data: any): Promise<any> {
+    const leaderPlanMembership = await PlanMembershipModel.dbCreateOnePlanMembership(data)
+    const friendsPlanMembership = await PlanMembershipModel.dbCreateManyPlanMembership(data)
```

Comment:
`const friendsPlanMembership` is not working at the moment because we haven't set up the frontend to send a proper array of users which are friends